text/0419-cdk-launchpads.md

```diff
@@ -0,0 +1,632 @@
+    scanOnPush: false, // default: true
+  }),
+});
+```
+
+The `cdk-lpads` command contains a validator that makes sure any changes you make to the bootstrap template are correct,
+and satisfy the minimal requirements needed by CDK in order to still deploy using those bootstrap resources.
+
+#### Adding new bootstrap resources
+
+You can also add new resources to the Launchpad that you want to be shared between all applications deployed into this environment.
+Application developers can then easily import these resources into their CDK applications.
+
+You can add the new resources using JSON/YAML files:
+
+```json
+{
+  "launchpad": {
+    "resources": {
+      "cfn": {
+        "Resources": {
+          "PermissionBoundaryPolicy": {
+            "Type": "AWS::IAM::ManagedPolicy",
+            "Properties": {
+              "PolicyDocument": {
+                "Statement": [
+                  {
+                    "Action": "iam",
+                    "Effect": "Deny",
+                    "Resource": "*"
+                  }
+                ],
+                "Version": "2012-10-17"
+              },
+              "Description": "The managed policy used as the permission boundary"
+            }
+          }
+        }
+      },
+      "export": {
+        "permissionBoundaryPolicy": {
+          "Ref": "PermissionBoundaryPolicy"
+        }
+      }
+    }
+  }
+}
+```
+
+And also with CDK, using the same CDK libraries that are used for applications:
+
+```ts
+import * as iam from 'aws-cdk-lib/aws-iam';
+
+const managedPolicy = new iam.ManagedPolicy(launchpad, 'Boundary', {
+  statements: [
+    new iam.PolicyStatement({
+      effect: iam.Effect.DENY,
+      actions: ['iam:'],
+      resources: [''],
+    }),
+  ],
+});
+// surface this resource to application developers using this Launchpad
+// (seee below)
+launchpad.export(managedPolicy);
+```
+
+#### IAM Permission Boundaries
+
+Launchpads allow setting a given Managed Policy as the Permissions Boundary to be used by every application deployed into a given environment.
+
+You can do it through JSON/YAML:
+
+```json
+{
+  "launchpad": {
+    "iam": {
+      "permissionBoundary": {
+        "Ref": "PermissionBoundaryPolicy"
+      }
+    }
+  }
+}
+```
+
+Or in CDK code:
+
+```ts
+launchpad.setPermissionBoundaryPolicy(managedPolicy);
+```
+
+### Importing the Launchpad in CDK code
+
+Once a Launchpad has been deployed by the administrators,
+application developers can import it using the
+[`cdk-import` tool](https://github.com/cdklabs/cdk-import)
+(whose functionality will be incorporated into the main CDK CLI in the future).
+
+Executing the following command:
+
+```shell
+$ cdk-import launchpad -l typescript
+```
+
+Will generate code in your project that you then use in your CDK app definition:
+
+```ts
+import * as cdk from 'aws-cdk-lib';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import Launchpad from '../build';
```

Comment:
Does this export a type of an object? Seems like it's an object, so use lowercase.

```suggestion
import launchpad from '../build';
```

Reply:
I thought lowercase was traditionally reserved for modules, but I don't have a strong opinion on this. Changed.